kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -137,14 +137,15 @@ <h4 class="mzp-c-menu-item-title">{{ _('Explore Our Help Articles') }}</h4>
               </svg>
               <h4 class="mzp-c-menu-item-title">{{ _('Mozilla Account') }}</h4>
               <p class="mzp-c-menu-item-desc">
-                {{ _('Find help for common issues and learn about the benefits a Mozilla Account provides.') }}
+                {{ _('Find help for common issues and learn about the benefits a Mozilla account provides.') }}
               </p>
             </a>
             <ul class="mzp-c-menu-item-list sumo-nav--sublist">
               <li><a href="{{ url('wiki.document', 'access-mozilla-services-firefox-account' )}}">{{ _('What is it?') }}</a></li>
-              <li><a href="https://accounts.firefox.com/reset_password">{{ _('Reset Password') }}</a></li>
-              <li><a href="{{ url('wiki.document', 'switching-devices') }}">{{ _('Sync My Data') }}</a></li>
               <li><a href="{{ url('products.product', 'mozilla-account')}}">{{ _('Get Help') }}</a></li>
+              <li><a href="{{ url('wiki.document', 'switching-devices') }}">{{ _('Sync My Data') }}</a></li>
+              <li></li>
+               <li><a href="https://accounts.firefox.com/reset_password">{{ _('Reset Password') }}</a></li>
```

Comment:
Super nit. Can we remove the extra space preceding this `<li>`.

Reply:
Done